src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs

```diff
@@ -449,6 +449,24 @@ public virtual Uri OptionsBlockedUrl
             }
         }
 
+        private string _optionsBlockedUrlText;
+
+        public virtual string OptionsBlockedUrlText
+        {
+            get
+            {
+                return _optionsBlockedUrlText;
+            }
+
+            set
+            {
+                _optionsBlockedUrlText = value;
+                OnPropertyChanged(nameof(OptionsBlockedUrlText));
+                OnPropertyChanged(nameof(OptionsBlockedUrl));
+                OnPropertyChanged(nameof(OptionsBlockedMessage));
```

Comment:
Wonder if these other property changed events need to be raised here. Mimicked the behavior from above to get feedback.